spec/ruby/optional/capi/exception_spec.rb

```diff
@@ -100,6 +100,26 @@
     end
   end
 
+  describe "rb_syserr_new" do
+    it "returns system error with custom message" do
+      const = Errno.const_get(Errno.constants.first)
```

Comment:
I'd rather pick a specific one than rely on constants ordering

Reply:
i did this because i wasn't sure how portable picking a named one would be.
looks like we are already using ENOENT in plenty of specs, i'll use that.